docs/tutorials/vagrant.rst

```diff
@@ -79,18 +79,14 @@ The Vagrant setup also enables port forwarding that allows your localhost to acc
     git clone https://github.com/rackhd/rackhd
     cd rackhd/example
 
+- Edit the Vagrantfile - delete the line 89: 'v.gui = true'
```

Comment:
If not, the quanta_d51 cannot set up successfully. The error message is about vm.gui.